src/Handlers/CardsPaymentMethodHandler.php

```diff
@@ -249,11 +257,19 @@ public function finalize(
         Request $request,
         SalesChannelContext $salesChannelContext
     ): void {
-        try {
+
+        //TODO this one will only handle 3DS2 but for 3DS1 we need the processResult
+        $this->paymentResponseHandler->finalize(
+            $transaction, $salesChannelContext
+        );
+        /*$context = $salesChannelContext->getContext();
+        $this->orderTransactionStateHandler->paid($transaction->getOrderTransaction()->getId(), $context);*/
+
+        /*try {
             $this->resultHandler->processResult($transaction, $request, $salesChannelContext);
         } catch (PaymentException $exception) {
-            throw new AsyncPaymentFinalizeException($exception->getMessage());
-        }
+            throw new AsyncPaymentFinalizeException($transaction->getOrderTransaction()->getId(), $exception->getMessage());
+        }*/
```

Comment:
Should we remove these comments?

Reply:
Let's keep it until we figure out how to test the 3DS1 flow and disabling the CSRF protection until it's fixed in shopware